modules/database/test/ioc/db/dbServerTest.c

```diff
@@ -163,8 +166,23 @@ MAIN(dbServerTest)
 
     testDiag("Checking server methods called");
     dbsr(0);
-    testOk(oneState == REPORT_CALLED, "dbsr called report()");
-
+    testOk(oneState == REPORT_CALLED, "dbsr called one::report()");
+
+    testDiag("Checking stats functionality");
+    testOk(dbServerStats("none", &ch, &cl) != 0, "Stats: unknown name rejected");
+    testOk(dbServerStats("no-routines", &ch, &cl) != 0, "Stats: no-routine rejected");
+    testOk(dbServerStats("one", &ch, &cl) == 0 && oneState == STATS_CALLED,
+        "dbServerStats('one') called one::stats()");
+    testOk(ch == 2 && cl == 1, "Stats: counts returned as expected");
+
+    ch = 0; cl = 0; oneState = NOTHING_CALLED;
+    testOk(dbServerStats(NULL, NULL, &cl) == 0 && oneState == STATS_CALLED,
+        "dbServerStats(NULL, &cl) called one::stats()");
+    testOk(dbServerStats(NULL, &ch, NULL) == 0 && oneState == STATS_CALLED,
+        "dbServerStats(NULL, &ch) called one::stats()");
+    testOk(ch == 2 && cl == 1, "Stats: counts returned as expected");
```

Comment:
I can imagine seeing:

```
not ok 42 Stats: counts returned as expected
```

and thinking... thank you, now what were they?

Reply:
Ack, I'm fixing the Codacy issue too.